modules/core/src/main/java/org/apache/ignite/configuration/CacheConfiguration.java

```diff
@@ -524,6 +526,7 @@ public CacheConfiguration(CompleteConfiguration<K, V> cfg) {
         sqlOnheapCache = cc.isSqlOnheapCacheEnabled();
         sqlOnheapCacheMaxSize = cc.getSqlOnheapCacheMaxSize();
         evtsDisabled = cc.isEventsDisabled();
+        isStatisticsEnabled = cc.isStatisticsEnabled();
```

Comment:
It will be nice to preserve alphabetical order
```
        /*
         * NOTE: MAKE SURE TO PRESERVE ALPHABETIC ORDER!
         * ==============================================
         */
```

Reply:
Let's fix this for the  `sqlOnHeapCache` and other properties as well.

Reply:
Fixed